packages/taquito/src/operations/smart-rollup-originate-operation.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  OperationContentsAndResult,
+  OperationContentsAndResultSmartRollupOriginate,
+  OpKind,
+} from '@taquito/rpc';
+import { Context } from '../context';
+import { Operation } from './operations';
+import {
+  FeeConsumingOperation,
+  ForgedBytes,
+  GasConsumingOperation,
+  RPCSmartRollupOriginateOperation,
+  StorageConsumingOperation,
+} from './types';
+
+/**
+ *
+ * @description SmartRollupOriginateOperation provides utility to fetch properties for Operation of kind UpdateConsensusKey
```

Comment:
last word to be SmartRollupOriginate

Reply:
sorry not being clear enough, I was trying to say update last word to be SmartRollupOriginate instead of UpdateConsensusKey